CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+-   The component `WebBookmark` has been renamed `LinkPreview`.
+-   Fix for `LinkPreview`: elements linked to optional props are no longer rendered when the related prop is not defined.
+
```

Comment:
Should be wrapped in sub changelog subsection (Added, Fixed, etc.) see https://keepachangelog.com/en/1.0.0/